### PR TITLE
[PLAT-719] Bring Google analytics up to date with Ember

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

GA custom dimension reporting for Embosf and osf.io is slightly different. This should make them identical.

## Changes

- custom dimension #1 is now capitalized
- custom dimension #2 now uses the resource type so "projects" or "components" are now "nodes"
- custom dimension #3 is no longer a bool, but either "public", "private" or "n/a"
## QA Notes

You should be able to set up a GA account and see the custom analytics been sent to it and see that they are seemly with the Ember analytics. Instructions on how to set up a GA account with custom dimensions are in the old PR for this [here](https://github.com/CenterForOpenScience/osf.io/pull/8140)

## Documentation

Basically a bug fix, no docs required.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-719

### CODE REVIEW

**Comment by sloria** [comment]
This is headed to staging3.
